.gitguardian.example.yml

```diff
@@ -36,3 +36,15 @@ secret:
   # Detectors to ignore.
   ignored-detectors: # default: []
     - Generic Password
+
+iac:
+  # Exclude files and paths by globbing
+  ignored-paths:
+    - 'tests/'
```

Comment:
```suggestion
    - 'tests/*'
```

Reply:
Both work, only `test/` is recursive and `test/*` is not. Why do you think adding `*` is best?

Reply:
I think it's good to be consistent with the secret example.